src/predictors/predictor-utils.cc

```diff
@@ -63,3 +63,16 @@ std::vector<nigiri::location> predictorUtils::get_stops_for_trip(nigiri::timetab
     }
     return stops;
 }
+
+void predictorUtils::deleteOldTripUpdates(std::unordered_set<std::string> currentTripIDs, transit_realtime::FeedMessage& outputFeed) {
```

Comment:
Can be declared const:
```suggestion
void predictorUtils::deleteOldTripUpdates(const std::unordered_set<std::string> currentTripIDs, transit_realtime::FeedMessage& outputFeed) {
```